qiskit/ignis/mitigation/measurement/circuits.py

```diff
@@ -16,43 +16,52 @@
 Measurement calibration circuits. To apply the measurement mitigation
 use the fitters to produce a filter.
 """
-
+from typing import List, Tuple
 from qiskit import QuantumRegister, ClassicalRegister, \
     QuantumCircuit, QiskitError
 from ...verification.tomography import count_keys
 
 
-def complete_meas_cal(qubit_list=None, qr=None, cr=None, circlabel=''):
+def complete_meas_cal(qubit_list: List[int] = None,
+                      qr: List[QuantumRegister] = None,
+                      cr: List[ClassicalRegister] = None,
+                      circlabel: str = ''
+                      ) -> Tuple[List[QuantumCircuit], List[str]
+                                 ]:
     """
     Return a list of measurement calibration circuits for the full
     Hilbert space.
 
     Each of the 2**n circuits creates a basis state
 
     Args:
-        qubit_list: A list of qubits to perform the measurement correction on,
-        if None and qr is given then assumed to be performed over the entire
-        qr. The calibration states will be labelled according to this ordering
+        qubit_list: A list of qubits to perform the measurement correction on.
+           If `None`, and qr is given then assumed to be performed over the entire
+           qr. The calibration states will be labelled according to this ordering (default `None`).
 
-        qr (QuantumRegister): A quantum register. If none one is created
+        qr: Quantum registers. If `None`, one is created (default `None`).
 
-        cr (ClassicalRegister): A classical register. If none one is created
+        cr: Classical registers. If `None`, one is created(default `None`).
 
         circlabel: A string to add to the front of circuit names for
-        unique identification
+            unique identification(default ' ').
 
     Returns:
-        A list of QuantumCircuit objects containing the calibration circuits
+        A list of QuantumCircuit objects containing the calibration circuits.
 
-        A list of calibration state labels
+        A list of calibration state labels.
 
     Additional Information:
         The returned circuits are named circlabel+cal_XXX
         where XXX is the basis state,
-        e.g., cal_1001
+        e.g., cal_1001.
 
         Pass the results of these circuits to the CompleteMeasurementFitter
-        constructor
+        constructor.
+
+    Raises:
+        QiskitError: if both `qubit_list` and `qr` are `None`.
+
```

Comment:
May be nice to add an Example at the end, see: https://github.com/Qiskit/qiskit-iqx-tutorials/blob/master/qiskit/advanced/ignis/4_measurement_error_mitigation.ipynb

code-block:
```
qr = qiskit.QuantumRegister(5)
qubit_list = [2,3,4]
meas_calibs, state_labels = complete_meas_cal(qubit_list=qubit_list, qr=qr, circlabel='mcal')
```
The output state_labels is: 
['000', '001', '010', '011', '100', '101', '110', '111']

Reply:
I won't add the example